Vocable/Features/Settings/TimingSensitivity/TimingSensitivityCollectionViewController.swift

```diff
@@ -123,14 +127,31 @@ class TimingSensitivityCollectionViewController: UICollectionViewController {
     }
     
     @objc private func handleDecreasingDwellTime(_ sender: UIButton) {
-        if AppConfig.selectionHoldDuration > 0.5 {
+        if AppConfig.selectionHoldDuration > minDwellTimeDuration {
             AppConfig.selectionHoldDuration -= 0.5
         }
+        
+        handleDwellTimeButtonRange()
     }
     
     @objc private func handleIncreasingDwellTime(_ sender: UIButton) {
-        if AppConfig.selectionHoldDuration < 5 {
+        if AppConfig.selectionHoldDuration < maxDwellTimeDuration {
             AppConfig.selectionHoldDuration += 0.5
         }
+        
+        handleDwellTimeButtonRange()
+    }
+    
+    private func handleDwellTimeButtonRange() {
+        guard let indexPath = dataSource.indexPath(for: .dwellTime) else { return }
+        guard let dwellTimeCell = collectionView.cellForItem(at: indexPath) as? DwellTimeCollectionViewCell else { return }
+        if AppConfig.selectionHoldDuration == minDwellTimeDuration {
+            dwellTimeCell.decreaseTimeButton.isEnabled = false
+        } else if AppConfig.selectionHoldDuration == maxDwellTimeDuration {
+            dwellTimeCell.increaseTimeButton.isEnabled = false
+        } else {
+            dwellTimeCell.decreaseTimeButton.isEnabled = true
+            dwellTimeCell.increaseTimeButton.isEnabled = true
+        }
```

Comment:
Can this be simplified?
```suggestion
        let currentDuration = AppConfig.selectionHoldDuration
        dwellTimeCell.decreaseTimeButton.isEnabled =  currentDuration > minDwellTimeDuration
        dwellTimeCell.increaseTimeButton.isEnabled = currentDuration < maxDwellTimeDuration
```

Reply:
Oh yeah that's so much cleaner, thanks